next.config.js

```diff
@@ -7,6 +7,9 @@ const nextConfig = {
       },
       {
         hostname: 'avatars.githubusercontent.com'
+      },
+      {
+        hostname: 'secure.gravatar.com'
```

Comment:
"avatar_url": "https://gitlab.com/uploads/-/system/user/avatar/4829338/avatar.png",

the avatar returned by GitLab starts with `gitlab.com`, sorry, I used your profile since idk if I have mine or not.

Should we also include this domain as well?

Reply:
Yeah sorry, my old profile pic (before I generated the new one) was a Gravatar image. I must have forgotten to test the new one. I'll update this branch in a bit